summingbird-online/src/main/scala/com/twitter/summingbird/planner/OnlinePlan.scala

```diff
@@ -27,6 +29,13 @@ class OnlinePlan[P <: Platform[P], V](tail: Producer[P, V]) {
   private val depData = Dependants(tail)
   private val forkedNodes = depData.nodes
     .filter(depData.fanOut(_).exists(_ > 1)).toSet
+
+  private def get[T <: AnyRef: ClassTag](dep: Producer[P, _]): Option[(String, T)] =
+    Options.getFirst[T](options, nameMap(dep))
+
+  private def getOrElse[T <: AnyRef: ClassTag](dep: Producer[P, _], default: T): T =
+    get[T](dep).map { case (namedSource, opts) => opts }.getOrElse(default)
```

Comment:
Maybe (_, t) => t?  namedSource and opts don't seem like quite the right labels here.